graphics/sprites/equipment/legs/skirt-female.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
 <sprite variants="135" variant_offset="1">
-  <imageset name="base" src="graphics/sprites/equipment/legs/skirt-male.png|W" width="64" height="64"/>
+  <imageset name="base" src="graphics/sprites/equipment/legs/skirt.png|W" width="64" height="64"/>
```

Comment:
@omattornot revert this change out as we need art as well #341 I've fixed the skirt here.